packages/core/src/app/shipping/Shipping.tsx

```diff
@@ -383,7 +383,7 @@ export function mapToShippingProps({
         isUpdatingCheckout() ||
         isCreatingCustomerAddress();
     const shouldShowMultiShipping =
-        hasMultiShippingEnabled && !methodId && shippableItemsCount > 1 && shippableItemsCount < 50;
+        hasMultiShippingEnabled && !methodId && shippableItemsCount > 1;
```

Comment:
@richapatel1510 As you can see here, multishipping mode has to be on to display this button while the shopper's sign-in state does not matter, hence the test should be inside the `when multishipping mode is on` group.
Maybe we can put the test at top of this group.

Reply:
In addtition, `hasMultiShippingEnabled` comes from the checkoutSettings, not a user action.

Reply:
@bc-peng In the group `when multishipping mode is on` the component is `component = mount(<ComponentTest {...defaultProps} isMultiShippingMode={true} />);` that is the case when the UI is in multishipping mode. That is not same as `hasMultiShippingEnabled` flag.

Reply:
I see your point, which would be valid if we always display 'Ship to multiple addresses' toggle as part of the shipping component.
However, the toggle does require `hasMultiShippingEnabled` to show up on the page.
Also, in order to set `isMultiShippingMode` to be true, `hasMultiShippingEnabled` needs to be true.

Reply:
`hasMultiShippingEnabled` is true by default in `packages/core/src/app/config/config.mock.ts` mock json.

Reply:
Then we should change it to `false` in order to keep it in line with our production environment. 😂

Reply:
@bc-peng Made the changes to other tests as well that were written based on default value of that flag.